plugin-server/package.json

```diff
@@ -33,7 +33,7 @@
         "prepublishOnly": "yarn build",
         "setup:dev:clickhouse": "cd .. && export DEBUG=1 PRIMARY_DB=clickhouse && python manage.py migrate_clickhouse",
         "setup:test:ee": "yarn setup:test:postgres && yarn setup:test:clickhouse",
-        "setup:test:postgres": "cd .. && (dropdb test_posthog || echo 'no db to drop') && createdb test_posthog && DATABASE_URL=postgres://localhost:5432/test_posthog DEBUG=1 python manage.py migrate",
+        "setup:test:postgres": "cd .. && (PGPASSWORD=posthog dropdb -h localhost -U posthog test_posthog || echo 'no db to drop') && PGPASSWORD=posthog createdb -h localhost -U posthog test_posthog && DATABASE_URL=postgres://posthog:posthog@localhost:5432/test_posthog DEBUG=1 python manage.py migrate",
```

Comment:
Just going to patch these as I find them... Probably should just get #7675 in as well

Reply:
Should we update deployment docs then? Seems like we need to create this username with a specific password, also to detail what permissions are needed.

Reply:
In #7730 I've proposed a slightly different approach so that people with env vars set get to keep them. And people without get sane defaults. 

I haven't extended that to plugin-server since it's being changed here